app-config-electron/README.md

```diff
@@ -0,0 +1,49 @@
+# App Config Electron
+
+Easily inject `app-config` values into Electron renderer processes. See `examples/electron-project` for an example project.
```

Comment:
FYI this readme will be used in NPM as well, where "see examples" won't make sense.

Reply:
Why won't it make sense in NPM? I just mean the examples folder in the repo

Reply:
Oh I see what you mean now. I changed it to a link